app/code/Magento/Checkout/view/frontend/web/js/model/cart/totals-processor/default.js

```diff
@@ -61,6 +61,8 @@ define([
         }).always(function () {
             // Stop loader for totals block
             totalsService.isLoading(false);
+            // Check for new session messages
+            customerData.reload('messages');
```

Comment:
Such changes will add one more request to the server on every totals calculation request. I think it's not optimal way for define extension points. Messages should be reloaded only in cases when needed (by additinal handlers in 3d party extensions).

Reply:
Fair enough.  #17127 takes care of that.

Only concern at this point is that there's no way for one module to know if another has decided to do that, but I think it'll be fine.  I'll retract this PR then